test/built-ins/DataView/prototype/getFloat16/resizable-buffer.js

```diff
@@ -0,0 +1,41 @@
+// Copyright (C) 2024 Kevin Gibbons. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-dataview.prototype.getfloat16
+description: Throws a TypeError if buffer is out-of-bounds
+features: [Float16Array, DataView, ArrayBuffer, resizable-arraybuffer]
+---*/
+
+assert.sameValue(
+  typeof ArrayBuffer.prototype.resize,
+  'function',
+  'implements ArrayBuffer.prototype.resize'
+);
```

Comment:
Is this necessary if we have the feature flag?

Reply:
I just copied this from the existing `getFloat32` test. I don't think it's necessary but I would prefer the new tests match existing ones.